### ISSUE TITLE
When connection to a Denon receiver fails, a message "Onkyo receiver can not found" appears

### ISSUE THREAD

**Issue by Joska59** [mkulesh/onpc#291]
@mkulesh

When I shutdown my receiver  ( while hole night, the receiver is power off,  no stand-by),  your app doesn't find the receiver again, when I switch on power again.
Then, a dialog appears with a hint, that under 192.168.x.y. no **onkyo**-receiver could be found. The check-box of **onyko** is marked, the other one of **denon/marantz** not.

It seems, that the app stores information about IP-adress, but not about the receiver-type.

After I'd turned on the receiver listening to music, the app finds the correct receiver, but under this circumstances, it's impossible to switch on the receiver by app.

**Comment by mkulesh** [comment]
The DRA-800H needs about 1.5 - 2 minutes to initialize network module (when the receiver is connected to the network using cable). For wireless connection the ready time may be more than 2 minutes. The app, however, has a connection timeout 10 seconds. For my DRA-800H, the app connects the receiver stable, if I just start the app 2 minutes after the receiver is powered on.

As a result, if you starts the app immediately after the receiver is powered, the connection will be refused with the message "Remote device xx.xx.xx.xx not responding". I you mean the German translation "Onkyo Gerät xx.xx.xx.xx antwortet nicht" of this message, you are right, this message is not consistent, I will improve it. 

Unfortunately I am not able to reproduce the behavior that you describe. After the message "Remote device xx.xx.xx.xx not responding" is shown, the app shall automatically open "Search" dialog. Within 2 minutes after power on, the Denon-Heos AVR usually appears in this dialog automatically. If a however close the "Search" dialog and manually open "Connect" dialog instead, the denon/marantz radio-box is selected as desired. Therefore, I have some questions:
- Which OS (iOS or Android) do you have?
- Which version of the app do you use?
- How your receiver is connected to the network: wired or wireless?

**Comment by mkulesh** [comment]
Developer notice: improve German translation:
```
    static const List<String> l_error_connection_no_response = [
        /*en*/ "Remote device %s not responding",
        /*ru*/ "Устойство %s не отвечает",
        /*de*/ "Onkyo Gerät %s antwortet nicht",
        /*fr*/ "Dispositif distant %s ne répond pas",
        /*pl*/ "Urządzenie zdalne %s nie odpowiada"];
```

**Comment by Joska59** [comment]
I use the android version 2.13.1 from google play store (payed version)
The receiver is connected by wire

**Comment by mkulesh** [comment]
And how stable does the app work, if you wait at least 2 minutes after power-on before connect to the receiver?

**Comment by Joska59** [comment]
I will check it a little bit later. Just now, the receiver was powered on
one hour ago, the controling, switchong on etc was totally okay.

Am 3. Januar 2024 19:30:57 schrieb Mikhail Kulesh ***@***.***>:

**Comment by Joska59** [comment]
Today, and I can't explain why, I did the same thing as before, first powered up the receiver, then after about 1 minute started the app and for the first time I received a reference to the correct IP address.  After clicking on the address, the receiver will be connected correctly.
 Great, that's exactly what I expected, but never found before.
 If it stays that way, the issue will happily be resolved quickly.
 Thanks.


Heute, und ich kann nicht erklären warum, habe ich das gleiche gemacht, wie bisher, erst den Receiver mit Strom versorgt, danach nach ca. 1 Minute die App gestartet und erhalte neu erstmalig einen Hinweis auf die korrekte IP Adresse. Nach Klick auf die Adresse wird der Receiver korrekt verbunden.
Super, genau so hatte ich es erwartet, aber bisher nie vorgefunden. 
Wenn es so bleibt, ist das Thema erfreulich schnell gelöst.
Danke.

**Comment by mkulesh** [comment]
Improved German translation if connection error occurs.

**Comment by mkulesh** [comment]
Released in version 2.13.2

**Comment by mkulesh** [comment]
Released in version 1.32
